### ISSUE TITLE
Add Default Comment Block Support

### ISSUE THREAD

**Issue by josh-beeman-homeward** [no-stack-dub-sack/apexdox-vs-code#50]
I would love to be able to load up the comment block standard that we utilize at my company so that we don't have to keep copy pasting it. Would love that support to work differently for the Method vs Class so we could define different defaults.

**Comment by no-stack-dub-sack** [comment]
This is a cool request! Time has been a little tight lately but I'll def add it to the backlog. Thanks!

**Comment by strotto** [comment]
+1 to this feature, mainly for Class level comments in my use case

**Comment by nwcm** [comment]
Yes to this! Being able to configure the default comment block for each type would be awesome. Similar to https://github.com/HugoOM/sfdx_autoheader

**Comment by no-stack-dub-sack** [comment]
This is related to #67. I think it could be made very flexible where defaults could be added at the tag level.

**Comment by jonathanwiesel** [comment]
Any update on this? Would be awesome to have this flexibility

**Comment by no-stack-dub-sack** [comment]
@jonathanwiesel Unfortunately with a lot of work and personal stuff going on lately and with relatively low demand from users I've had to de-prioritize taking on enhancement requests for this project, though I am def open to PRs and will continue to fix any major bugs.

I'm not saying this will never get done, but it's on the back burner for now unless someone is willing to work on a PR.

Long term, I'm considering turning over the maintenance of the project to someone else in order to ensure better support for whatever user base there is (which is hard to determine as the repo does not get a lot of traffic and it's impossible to tell from the download count how many people are actively using it. If you like the project, consider starring it if you haven't already! Stars are at least one metric that can help me judge demand, but despite moderate activity over the years the project has amassed very few.).
